conf/application.conf

```diff
@@ -184,5 +184,6 @@ microservice {
 
 submissionStubUrl = "http://localhost:9303"
 useBusinessDetailsIFPlatform = true
+useRepaymentHistoryDetailsIFPlatform = true
```

Comment:
we would need to have next entry in the configs as well:
authorization-token-1771